File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    } while (h < 1 || h > 8);

    for (int i = 1; i <= h; i++)
    {

        add_space(h - i);
        add_brick(i);
        printf(" ");
        printf(" ");
        add_brick(i);
        add_space(h - i);

    }
}

void add_space(int n)
{
    for (int i = 0; i < n; i++)
    {
        printf(" ");
    }
}

void add_brick(int n)
{
    for (int i = 0; i < n; i++)
    {
        printf("#");
    }
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

height = get_int("Height: ")
while height < 1 or height > 8:
    height = get_int("Height: ")
for i in range(1, height+1):
    for x in range(height-i):
        print(" ", end="")
    for y in range(i):
        print("#", end="")
    for h in range(2):
        print(" ", end="")
    for k in range(i):
        print("#", end="")
    print("")
